### PR TITLE
github: add release-note description format to pull request template

### PR DESCRIPTION
**Pull request by CadmusJiang**
Signed-off-by: Cadmus <CadmusJiang@gmail.com>

### What problem does this PR solve?
Help Tichi to realize the information collection function of release note.

### What is changed and how it works?
Add a release note reference to pull_ request_template.md
### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Code changes

- Has configuration change
- Has HTTP API interfaces change (Don't forget to [add the declarative for API](https://github.com/tikv/pd/blob/master/docs/development.md#updating-api-documentation))
- Has persistent data change

Side effects

- Possible performance regression
- Increased code complexity
- Breaking backward compatibility

Related changes

- PR to update [`pingcap/docs`](https://github.com/pingcap/docs)/[`pingcap/docs-cn`](https://github.com/pingcap/docs-cn):
- PR to update [`pingcap/tidb-ansible`](https://github.com/pingcap/tidb-ansible):
- Need to cherry-pick to the release branch

### CODE REVIEW

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -50,5 +50,7 @@ Related changes
 ### Release note
 
 <!-- A bugfix or a new feature needs a release note. If there is no need release note, just uncomment the below line. -->
-
+```release-note
+    Some release note.
```
Comment (hi-rustin) [inline .github/pull_request_template.md:54]:
```suggestion
Please add a release note.

If you don't think this PR needs a release note then fill it with None.
```

**Review by hi-rustin — CHANGES_REQUESTED** [review]

**Review by nolouch — APPROVED** [review]

File: .github/pull_request_template.md [hunk 2]
```diff
@@ -50,5 +50,9 @@ Related changes
 ### Release note
 
 <!-- A bugfix or a new feature needs a release note. If there is no need release note, just uncomment the below line. -->
+```release-note
+Please add a release note.
 
+If you don't think this PR needs a release note then fill it with None.
+```
 <!-- - No release note -->
```
Comment (hi-rustin) [inline .github/pull_request_template.md:58]:
```suggestion
```

**Review by hi-rustin — COMMENTED** [review]
Better to remove it.

**Pull request merged**